### PR TITLE
Change duckduckgo's logo to its dark form

### PR DESCRIPTION
**Pull request by AndrewDaGuy**
Home page is more readable ("DuckDuckGo" text is in light instead of dark)
and yes I did yoink the CSS

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -2425,7 +2426,20 @@ CSS
 .justify-content-center.align-items-center.flex-lg-column.flex-md-row.card-2.card > .card-body > p,
 .justify-content-center.align-items-center.flex-lg-column.flex-md-row.card-3.card > .card-body > p,
 .js-hiring-open-pos.join__btn.mt-5.btn.col-sm-auto.col{
-  color: var(--darkreader-neutral-background) !important;
+    color: var(--darkreader-neutral-background) !important;
+}
+
+.logo_homepage {
+    background-image: url("assets/logo_homepage.alt.v108.svg") !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:2433]: Please use a absolute URL like `https://duckduckgo.com/assets/logo_homepage.alt.v108.svg`.

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -2425,7 +2426,20 @@ CSS
...
+}
+
+.header__logo {
+    background-image: url("assets/logo_header.alt.v108.svg") !important;
+}
+
+.header--html .header__logo {
+    background: no-repeat center url("assets/logo_header.alt.v108.svg") !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:2441]: Double property?
Please use a absolute URL like `https://duckduckgo.com/assets/logo_homepage.alt.v108.svg`.

**Review by Gusted — CHANGES_REQUESTED** [review]

File: src/config/dynamic-theme-fixes.config [hunk 3]
```diff
@@ -2434,6 +2444,7 @@ IGNORE INLINE STYLE
 
 ================================
 
+
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:2447]: Remove the new line here.

File: src/config/dynamic-theme-fixes.config [hunk 4]
```diff
@@ -2425,7 +2426,16 @@ CSS
...
+    color: var(--darkreader-neutral-background) !important;
+}
+
+.logo_homepage {
+    background-image: url("https://duckduckgo.com/assets/logo_homepage.alt.v108.svg") !important;
+}
+
+.header--html .header__logo, .header__logo {
+    background: no-repeat center url("https://duckduckgo.com/assets/logo_header.alt.v108.svg") !important;
+    background: no-repeat center/auto 36px url("https://duckduckgo.com/assets/logo_header.alt.v108.svg") !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:2438]: Once again, why double property here?

**Review by Gusted — CHANGES_REQUESTED** [review]

File: src/config/dynamic-theme-fixes.config (hunk 4 above)
Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:2438]: Don't mark it as resolved when I still don't understand why a double property is here.

Comment (AndrewDaGuy) [reply src/config/dynamic-theme-fixes.config:2438]: There was a double property there because I "yoinked" the css from ddg 
> and yes I did yoink the CSS

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:2438]: You can remove the top property it doesn't add anything to the fix.

File: src/config/dynamic-theme-fixes.config [hunk 5]
```diff
@@ -2425,7 +2426,15 @@ CSS
...
 .justify-content-center.align-items-center.flex-lg-column.flex-md-row.card-3.card > .card-body > p,
 .js-hiring-open-pos.join__btn.mt-5.btn.col-sm-auto.col{
-  color: var(--darkreader-neutral-background) !important;
+    color: var(--darkreader-neutral-background) !important;
+}
+
+.logo_homepage {
+    background-image: url("https://duckduckgo.com/assets/logo_homepage.alt.v108.svg") !important;
+}
+
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:2435]: Remove new line

File: src/config/dynamic-theme-fixes.config [hunk 6]
```diff
@@ -2425,7 +2426,15 @@ CSS
 .justify-content-center.align-items-center.flex-lg-column.flex-md-row.card-2.card > .card-body > p,
 .justify-content-center.align-items-center.flex-lg-column.flex-md-row.card-3.card > .card-body > p,
 .js-hiring-open-pos.join__btn.mt-5.btn.col-sm-auto.col{
-  color: var(--darkreader-neutral-background) !important;
+    color: var(--darkreader-neutral-background) !important;
+}
+
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:2431]: Remove new line

**Review by Gusted — CHANGES_REQUESTED** [review]

**Pull request merged**